src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Models/DeploymentTemplate.cs

```diff
@@ -185,8 +182,7 @@ internal static string ToJsonString(this DeploymentTemplate template)
                             AdminUserEnabled = adminUserEnabled != null ? adminUserEnabled.Value : false,
                             StorageAccount = new StorageAccount
                             {
-                                Name = storageAccountName,
-                                AccessKey = $"[listKeys(resourceId('{storageAccountResourceGroup}', 'Microsoft.Storage/storageAccounts', '{storageAccountName}'), '{StorageApiVersion}').keys[0].value]"
+                                Id = $"[resourceId('Microsoft.Storage/storageAccounts', '{storageAccountId}')]"
```

Comment:
Seems no need to reconstruct an `id` from an `id`, can we just use `[parameters('storageAccountId')]` here?

Reply:
actually we don't need this method now. We are using Registry.Update now.